plugins/assertible/lib/samson_assertible/samson_plugin.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module SamsonAssertible
+  class Engine < Rails::Engine
+  end
+
+  class Notification
+    class << self
+      def deliver(deploy)
+        return unless service_key.present?
+        return unless deploy_token.present?
+        return unless deploy.stage.notify_assertible?
```

Comment:
fail-fast would be nice here ... if the stage is set to `notify_assertible ` then blow up if key/token is not set ?
`ENV.fetch('ASSERTIBLE_DEPLOY_TOKEN')`